spec/furniture/marketplace/delivery_spec.rb

```diff
@@ -1,7 +1,10 @@
 require "rails_helper"
 
 RSpec.describe Marketplace::Delivery, type: :model do
-  subject(:delivery) { described_class.new }
+  subject(:delivery) { build(:marketplace_delivery, marketplace: marketplace, delivery_area: delivery_area) }
+
+  let(:marketplace) { build(:marketplace) }
+  let(:delivery_area) { nil }
 
   it { is_expected.to belong_to(:marketplace) }
   it { is_expected.to belong_to(:shopper) }
```

Comment:
Might be worth adding expectations for the new `belongs_to`, and the new validation you added.